wiki/Music_theory/Layer/en.md

```diff
@@ -0,0 +1,29 @@
+---
+tags:
+  - musical layer
+  - sound layer
+---
+
+# Layer
+
+A **musical layer** is a part of the song which contains features with a common aspect between them. A significant part of [mapping](/wiki/Beatmapping) in every [game mode](/wiki/Game_mode) revolves around following different kinds of layers (percussion, vocals, sound patterns) consistently, and switching between them for the purpose of emphasis and contrast.
+
+## Description
+
+Typically, a single layer in a musical piece represents a group of instruments of the same family or [range](https://en.wikipedia.org/wiki/Range_(music)), or — alternatively — voices of the [same type](https://en.wikipedia.org/wiki/Voice_type). Since layers are abstract, they can be limited to just a single instrument or kind of sound, depending on the context they are discussed in. Other criteria for layer subdivision include [rhythm](https://en.wikipedia.org/wiki/Rhythm) patterns, [melody](https://en.wikipedia.org/wiki/Polyphony), [harmony](https://en.wikipedia.org/wiki/Harmony), or [pitch](https://en.wikipedia.org/wiki/Pitch_(music)).
+
+## Mapping
+
+In osu! as a game, song layers are represented through different [patterns](/wiki/Beatmap/Pattern) and sections of the beatmap. While picking a layer to follow is usually up to a beatmap author's discretion, the choices are usually the most obvious and recognisable ones:
+
+- Percussion, such as drums, hi-hats, or other similar instruments providing the [beat pattern](/wiki/Music_theory/Beat)
+- Instruments forming the main melody pattern(s)
+- Vocal party
```

Comment:
"beat pattern" is a bit of a nebulous connection

as mentioned, the rhythms of these is taken into consideration and the one followed directly corresponds to where to place hit objects on the timeline, as well as which hit object to use

Reply:
I meant the sounds which form the tempo (1/1 beats and smaller notes that surround them). I can change "beat pattern" to something more direct to clarify that.

Reply:
"rhythm" is the more direct word here, and i don't think i understand your explanation, since the "beat" is just a repeating pulse

you're not mapping to a "tempo" (or in other words mapping to the "speed of the song", whatever that would mean), but the various *rhythms* formed by the percussion, other instruments, vocals, etc.

it's the terminology that i'm fundamentally concerned about

Reply:
fair enough, removed